tests/testthat/test-post-action-tailor.R

```diff
@@ -109,91 +109,32 @@ test_that("postprocessor fit aligns with manually fitted version (no calibration
 
 test_that("postprocessor fit aligns with manually fitted version (with calibration)", {
```

Comment:
The diff here is kinda funky, but these changes just adjust to the new interface in

> postprocessor fit aligns with manually fitted version (with calibration)

and remove the test block

> postprocessor fit uses correct data (with calibration, non-default `prop`)